frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -421,8 +511,51 @@ export const ClusterServiceVersionList: React.SFC<ClusterServiceVersionListProps
   );
   const NoDataEmptyMsg = () => <MsgBox title="No Operators Found" detail={noDataDetail} />;
 
-  return (
+  // remove duplicate global operator entries
+  const filteredData =
+    data.length > 0
+      ? data.filter((source) => {
+          if (source === undefined || source === null) {
+            return null;
+          }
+          if (source.kind !== 'ClusterServiceVersion') {
```

Comment:
Instead of a string, can you reference the Model kind?

Reply:
I'll push an updated version after responding to the other feedback; let me know if you want a different approach once you can see it.